lib/uwb/include/uwb/protocols/fira/FiraDevice.hxx

```diff
@@ -602,6 +612,8 @@ constexpr std::array<UwbApplicationConfigurationParameterType, 5> UwbApplication
     UwbApplicationConfigurationParameterType::RangeDataNotificationProximityFar,
 };
 
+constexpr auto MAX_CONTROLEES = 8;
```

Comment:
There is a place somewhere in one of the specs where it says the max number of controlees could be greater than 8 (@abeltrano do you remember where?). This came up in one of my previous PRs, otherwise this constant would already exist below line 60 (below `constexpr uint8_t MinimumNumberOfControlees = 1;`)

Reply:
That's right, it's in CSML Table 7, UWB Configuration Parameters from Controller:

![image](https://user-images.githubusercontent.com/2082148/233706863-b2cfb6fc-8def-4e57-9758-a67dab60d5ba.png)

Reply:
Also, if we do need some kind of constant, please name it per style guidelines using PascalCase. Since this value is a little weird, adding some clarifying comments to the declaration/definition will definitely help:

```suggestion
/**
 * The minimum number of maximum controlees a particular implementation must handle. The maximum number of controlees may be higher if the implementation can support it.
 */
constexpr auto MaxControleesMin = 8;
```

(I'm not sure if the variable name I used above is the best, but you get the idea)

Reply:
Implementation can support more than 8 anchors, but the UCI spec doesn't mention this. Should we just follow the UCI spec since the command we're implementing is described there?

Reply:
I also recommend moving this constant to where the other similar constants are (around line 60 in my previous comment)

Reply:
> Implementation can support more than 8 anchors, but the UCI spec doesn't mention this. Should we just follow the UCI spec since the command we're implementing is described there?

I would probably just remove the code that's enforcing this maximum (and thus, remove this constant too). We should just let the driver handle it, which will be forwarded to the device which has the impl and it will indicate whether it's valid or not.